Spejderneslejr/spejderneslejrdk
# Show updated timestamp on articles

Show when a node/article is last updated but only when the time changed is two minutes after creation.

When a node is created and thus saved, it also generates a timestamp for `getChangedTime()`. You would expect it to be equal to `getCreatedTime()` but that is not the case. `getChangedTime()` is actually a little off compared to `getCreatedTime()`. Therefore, we add a check to only show the `updated_timestamp` if a specific time has passed which is the `$threshold` variable.

**How to test**
1. [Go to platform env](https://www.pr-261-tpwheba-kigwipdp3tsiq.eu-4.platformsh.site/)
2. If you want, you can translate the "Last updated @updated_timestamp" string (not necessary) to test it in en/da
2. See existing nodes have the "Last updated" under headlines.
3. Create a new article and see that the "last updated" is not visible. If you wait two minutes (the `$threshold`) and update the article, you should see the "Last updated"

<img width="980" alt="image" src="https://user-images.githubusercontent.com/68547893/154086535-b904aa58-cb66-4ef8-ab26-bed9a07fde48.png">
